cpu/mips_pic32mx/p32mx470f512h/p32mx470f512h.S

```diff
@@ -0,0 +1,2732 @@
+ #-------------------------------------------------------------------------
+ # MPLAB XC Compiler -  PIC32MX470F512H processor definition module
+ #
+ # Copyright (c) 2016, Microchip Technology Inc. and its subsidiaries ("Microchip")
+ # All rights reserved.
+ #
```

Comment:
This also look as a vendor thing... I think it's better to pu it into the vendor folder.